sources/packages/web/src/App.vue

```diff
@@ -1,4 +1,5 @@
 <template>
+  <environment-banner />
```

Comment:
Please move the banner inside `<v-app>` otherwise the component will not be under the scope of Vuetify